workers/dockerfiles/Dockerfile.transcriptome

```diff
@@ -75,9 +50,22 @@ RUN curl -k -L https://github.com/COMBINE-lab/salmon/archive/${SALMON_COMMIT_HAS
 ENV PATH /home/user/salmon-${SALMON_COMMIT_HASH}/bin:${PATH}
```

Comment:
The installation steps for salmon seem a little out-dated. The latest version is 0.9.1:
https://github.com/COMBINE-lab/salmon/releases
Since `-DCMAKE_INSTALL_PREFIX=/usr/local` option is used in cmake, the binary will be installed in /usr/local/bin, so line 50 is not needed.

Reply:
I agree that we should update Salmon to the latest version.